packages/manager/apps/public-cloud/src/index.routes.js

```diff
@@ -41,6 +41,24 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
     translations: { value: ['.'], format: 'json' },
   });
 
+  $stateProvider.state('app.userContracts', {
+    url: 'user-contracts',
+    redirectTo: ($transition$) => {
+      const { contracts } = $transition$.params();
+      return contracts ? false : 'app';
+    },
+    component: 'userContracts',
+    params: {
+      contracts: null,
+    },
+    translations: { value: ['.'], format: 'json' },
```

Comment:
Should we import translations from `packages/manager/apps/public-cloud/src/translations` for this state ?